### ISSUE TITLE
Bootstrap environment variables for automated install are not applied when starting without blueprints

### ISSUE THREAD

**Issue by mittler-works** [goauthentik/authentik#7546]
**Describe the bug**
When I start authentik from green field with an empty `/blueprints` directory, the bootstrap variables `AUTHENTIK_BOOTSTRAP_EMAIL`, `AUTHENTIK_BOOTSTRAP_PASSWORD` and `AUTHENTIK_BOOTSTRAP_TOKEN` don't seem to be applied.

**To Reproduce**
1. Follow the official instructions until before you start authentik https://goauthentik.io/docs/installation/docker-compose 
2. Modify the `docker-compose.yml` file and add another volume for the worker and the server like this: `./empty:/blueprints:ro` *
3. Add `AUTHENTIK_BOOTSTRAP_EMAIL`, `AUTHENTIK_BOOTSTRAP_PASSWORD` and `AUTHENTIK_BOOTSTRAP_TOKEN` with proper values to your `.env` file
4. Now start authentik

_\* For whatever reason, it fills the empty directory with the standard blueprints when using a volume instead of a bind mount._

**Expected behavior**
An completely empty authentik without any flows or stuff from blueprints, BUT with an akadmin user with given email, password and an api token.

**Actual behavior**
An completely empty authentik, but with none of the bootstrap variables applied.

**Logs**
Database output for users and tokens:

```
authentik=# select * from authentik_core_user ;
 id |                 password                  | last_login |                  username                   | first_name | last_name | email | is_active |          date_joined          |                 uuid                 |                        name                        |     password_change_date      | attributes |          path           |           type           
----+-------------------------------------------+------------+---------------------------------------------+------------+-----------+-------+-----------+-------------------------------+--------------------------------------+----------------------------------------------------+-------------------------------+------------+-------------------------+--------------------------
  1 | !VdRM9WWDzIKE21Aua3HQmQVQ3nXt60jEYgzIsmno |            | AnonymousUser                               |            |           |       | t         | 2023-11-13 15:08:58.435892+00 | 021e0f27-b040-4bc9-9568-3b7708ebb2e6 |                                                    | 2023-11-13 15:08:58.436603+00 | {}         | users                   | internal
  2 | !zYv94md0CaMecjBsY1BENhJTvJbH1XVZmQe1OCwj |            | ak-outpost-319ef80b14d94e04b14cc1fb09322312 |            |           |       | t         | 2023-11-13 15:09:12.988442+00 | 709772b6-3c23-4fc1-8955-8ab31d4599e4 | Outpost authentik Embedded Outpost Service-Account | 2023-11-13 15:09:12.988766+00 | {}         | goauthentik.io/outposts | internal_service_account
(2 rows)
```

```
authentik=# select * from authentik_core_token ;
 token_uuid | expires | expiring | description | user_id | intent | identifier | key | managed 
------------+---------+----------+-------------+---------+--------+------------+-----+---------
(0 rows)
```

**Version and Deployment (please complete the following information):**

-   authentik version: 2023.10.3
-   Deployment: docker-compose

**Additional context**
The setup worked perfectly fine until (including) 2023.6 versions. The issue occurs from version 2023.8 onwards.

**Comment by mittler-works** [comment]
Ok, I guess this is why the old way is not working anymore: https://github.com/goauthentik/authentik/pull/6433

This has an unfortunate side effect: Authentik can no longer be seeded by only using the official Authentik terraform provider (https://github.com/goauthentik/terraform-provider-authentik). This breaks the automated install in environments that are based on these bootstrap stuff in combination with the terraform provider and that do not use blueprints.

I guess there's a reason for this change, but maybe there's also another solution for this? I wouldn't actually like to mix Blueprints and terraform provider stuff as this results in being a potential source of (human) issues in my opinion.

**Comment by BeryJu** [comment]
While not as simple, you can remove all other blueprints aside from the bootstrap one

(For context, the main reason why this was changed was because the code using the _bootstrap environment variables was in django migrations which makes it quite annoying to maintain, and it also won't ever re-run in the future, which is also one of the main reason why all the default flows and such are blueprints)

**Comment by mittler-works** [comment]
Thanks for the reply. I completely understand why the code should be removed from the migrations.
In my opinion, however, it is not optimally placed in the blueprints either. This bootstrap stuff does not need to ever re-run in future, does it? But to be honest, I can't think of a better solution off the top of my head.

For docker-compose I've solved it by simply having a dir with `system/bootstrap.yaml` in it and bind mount it to `/blueprints`
For K8s I've solved it by using two mounts: an emptyDir volume on `/blueprints` and a configMap volume containing the blueprint on `/blueprints/system/bootstrap.yaml`.

Not the most beautiful, but a working solution.
